### ISSUE TITLE
Feature request: OpenAI Image generation

### ISSUE THREAD

**Issue by skobylite** [intellibar/main#18]
Image generation within intellibar would be a cool feature using the openAI API.

**Comment by astoilkov** [comment]
Yeah, we might do that. We will think about it. What's your use case? What type of images are you going to generate? Thanks!
